### ISSUE TITLE
[Accessibility]: Code monitoring create/edit page ARC checklist issues

### ISSUE THREAD

**Issue by limitedmage** [sourcegraph/sourcegraph#35750]
### Audit type

ARC Toolkit

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34195

### Problem description

The following issues need to be fixed on the [Code Monitoring create/edit page](https://sourcegraph.com/code-monitoring/new):

- [ ] The code monitoring logo should not have an `aria-label` on the `a` tag and have an `aria-label` on the `svg` tag reads "Code Monitoring" 
  ![image](https://user-images.githubusercontent.com/206864/169405705-61497686-e06b-4037-bcbb-53ca1e421a0e.png)
- [ ] The heading "Manage/create code monitor" should be h2 instead of h1 and "Trigger"/"Actions" headers should be h3 instead of h2. Also, the headings inside *open* cards (both trigger and actions) should be h4.
  ![CleanShot 2022-05-19 at 14 16 20](https://user-images.githubusercontent.com/206864/169406112-53fb43db-fcee-4f9c-ae81-d1af68ae95f5.png)
- [ ] The owner should be labelled properly. Dropdown menu `select` tag for owner should have an ID and the `label` before it should have a `for` property to that ID. Also remove the `aria-label` for that `select` as it should no longer be needed.
  ![image](https://user-images.githubusercontent.com/206864/169406348-33f721d0-8c87-403d-8b31-c4f12d55ac5c.png)
- [ ] Remove `aria-label` from closed trigger & action cards. Add a visually hidden text instead before the card's title (eg. "Send email notifications") that reads "Edit trigger:" or "Edit action:". 
  ![CleanShot 2022-05-19 at 14 22 45](https://user-images.githubusercontent.com/206864/169407055-63a1db2e-8f38-4e8d-96a6-215042c9df96.png)
- [ ] SVG icons in trigger query checklist should be aria-hidden
  ![CleanShot 2022-05-19 at 14 25 52](https://user-images.githubusercontent.com/206864/169407460-df6ad12d-2b47-4edd-9126-253dab49f552.png)
- [ ] In the trigger card, icon next to "Preview results" should have `aria-label="Open in new window"`
  ![CleanShot 2022-05-19 at 14 27 36](https://user-images.githubusercontent.com/206864/169407627-c57362ab-b796-4adc-9044-6815a0070897.png)
- [ ] In the Slack notifications card (NOT the webhooks card), rename label "Webhook URL" to "Slack webhook URL"
  ![CleanShot 2022-05-19 at 14 29 37](https://user-images.githubusercontent.com/206864/169407983-783b6e87-4d37-41e5-b691-0899d561990c.png)
- [ ] Remove title attribute from all toggles and ensure that `aria-labelledby` is pointing to a valid element (it is incorrect in some of them).
  ![CleanShot 2022-05-19 at 14 36 08](https://user-images.githubusercontent.com/206864/169408773-fa676531-7b11-4fbb-ab0b-0436ac7edc86.png)

  

  

### Expected behavior

All issues are fixed as described above

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by GitStart. I have assigned the label 'gitstart' and added the assignee 'gitstart-sourcegraph'.

**Comment by sourcegraph-bot-2** [comment]
Heads up @benvenker @lguychard - the "team/search-product" label was applied to this issue.

**Comment by gitstart-sourcegraph** [comment]
Hi @limitedmage 
Wondering if this requirement: `The code monitoring logo should not have an aria-label on the a tag and have an aria-label on the svg tag reads "Code Monitoring"` applies to other similar use cases of page header breadcrumb with `Icon` as first breadcrumb element?

`Assumption:` Use cases with only icon as breadcrumb elements, should have `aria-label` binded to the svg icon instead of wrapper?
